core/players-handbook/race-tiefling.xml

```diff
@@ -0,0 +1,96 @@
+			<!-- (removed in Errata v3.0) -->
+			<!-- <p class="indent"><strong><em>Alignment.</em></strong> Tieflings might not have an innate tendency toward evil, but many of them end up there. Evil or not, an independent nature inclines many tieflings toward a chaotic alignment.</p> -->
+			<p class="indent"><strong><em>Size.</em></strong> Tieflings are about the same size and build as humans. Your size is Medium.</p>
+			<p class="indent"><strong><em>Speed.</em></strong> Your base walking speed is 30 feet.</p>
+			<p class="indent"><strong><em>Darkvision.</em></strong> Thanks to your infernal heritage, you have superior vision in dark and dim conditions. You can see in dim light within 60 feet of you as if it were bright light, and in darkness as if it were dim light. You can’t discern color in darkness, only shades of gray.</p>
+			<p class="indent"><strong><em>Hellish Resistance.</em></strong> You have resistance to fire damage.</p>
+			<p class="indent"><strong><em>Infernal Legacy.</em></strong> You know the thaumaturgy cantrip. Once you reach 3rd level, you can cast the hellish rebuke spell, as a 2nd-level spell, you must finish a long rest to cast the spell again with the trait. Once you reach 5th level, you can also cast the darkness spell, you must finish a long rest to cast the spell again with the trait. Charisma is your spellcasting ability for these spells.</p>
+			<p class="indent"><strong><em>Languages.</em></strong> You can speak, read, and write Common and Infernal.</p>
+		</description>
+		<sheet display="false" />
+		<setters>
+			<set name="names" type="male">Akmenos, Amnon, Barakas, Damakos, Ekemon, Iados, Kairon, Leucis, Melech, Mordai, Morthos, Pelaios, Skamos, Therai</set>
+			<set name="names" type="female">Akta, Anakis, Bryseis, Criella, Damaia, Ea, Kallista, Lerissa, Makaria, Nemeia, Orianna, Phelaia, Rieta</set>
+			<set name="names" type="virtue">Art, Carrion, Chant, Creed, Despair, Excellence, Fear, Glory, Hope, Ideal, Music, Nowhere, Open, Poetry, Quest, Random, Reverence, Sorrow, Temerity, Torment, Weary</set>
+			<set name="names-format">{{name}} “{{virtue}}”</set>
+			<set name="height" modifier="2d8">4'9"</set>
+			<set name="weight" modifier="2d4">110 lb.</set>
+		</setters>
+		<rules>
+			<stat name="intelligence" value="1" requirements="!(ID_INTERNAL_GRANTS_TIEFLING_SUBRACE|ID_WOTC_TCOE_OPTION_CUSTOMIZED_ASI)" />
+			<stat name="charisma" value="2" requirements="!(ID_INTERNAL_GRANTS_TIEFLING_SUBRACE|ID_WOTC_TCOE_OPTION_CUSTOMIZED_ASI)"/>
+			<select type="Ability Score Improvement" name="Custom Ability Score Improvement +2 (Tiefling)" supports="Custom Ability Score Increase 2" requirements="!ID_INTERNAL_GRANTS_TIEFLING_SUBRACE,ID_WOTC_TCOE_OPTION_CUSTOMIZED_ASI" />
+			<select type="Ability Score Improvement" name="Custom Ability Score Improvement +1 (Tiefling)" supports="Custom Ability Score Increase 1" requirements="!ID_INTERNAL_GRANTS_TIEFLING_SUBRACE,ID_WOTC_TCOE_OPTION_CUSTOMIZED_ASI" />
```

Comment:
i would also say no reason to remove that portion from Variant and then re-add it in the Variant... like why? It does the same thing either way.

Reply:
The MToF subraces do the same thing.